packages/perseus/src/perseus-types.ts

```diff
@@ -174,7 +174,7 @@ export const ItemExtras = [
 ] as const;
 export type PerseusAnswerArea = Record<(typeof ItemExtras)[number], boolean>;
 
-type Widget<Type extends string, Options> = {
```

Comment:
This was really confusing. We had two types named `Widget` in our codebase: one represented the React component and the other the widget options. Renaming this will hopefully make things clearer.